operator-pipeline-images/tests/entrypoints/test_github_add_comment.py

```diff
@@ -0,0 +1,87 @@
+from logging import raiseExceptions
+import pathlib
+from unittest.mock import MagicMock, patch
+
+from operatorcert.entrypoints import github_add_comment
+import pytest
+
+GITHUB_HOST_URL = "https://api.github.com"
+PARENT_DIR = pathlib.Path(__file__).parent.resolve()
+COMMENT_PATH = PARENT_DIR.joinpath("../data/comment.txt")
+REQUEST_URL = "https://github.com/redhat-openshift-ecosystem/operator-pipelines/pull/252"
+
+
+@patch("operatorcert.entrypoints.github_add_comment.github.post")
+def test_github_add_comment_POST(mock_post: MagicMock) -> None:
```

Comment:
Please keep the function name lowercase. Also, update other function names in this file.